src/App.js

```diff
@@ -0,0 +1,78 @@
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+import { Route, Routes } from 'react-router';
+import './App.css';
+
+import { useLoading } from '@agney/react-loading';
+import NavBar from './Components/NavBar/NavBar';
+import { useAuth, useCart} from './Context/context';
+import CartPage from './Pages/CartPage/CartPage';
```

Comment:
Use `index.js` in each folder to export everything so that all components can be imported in a single statement

Reply:
Thanks for the Advice! Implemented this.